packages/codemods/tsup.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig([
```

Comment:
We are currently not using `@suspensive/tsup` and have instead declared the options directly. In the future, we plan to define the options in `@suspensive/tsup` and standardize them with the scriptOptions used in `@suspensive/react-query.`